src/lib.rs

```diff
@@ -40,7 +40,7 @@
 #![feature(core_intrinsics)]
 
 
-static SIZE_ALLOC_TABLE: usize = (1024 * 1024) * 2; // 2KB
+static SIZE_ALLOC_INFO: usize = (1024 * 1024) * 2; // 2KB
```

Comment:
I still don't think this is 2KiB!

Reply:
nothing to see here... :eyes:

Reply:
He was just testing you Laurie. Well done, you passed 😆